sql_generators/mobile_kpi_support_metrics/__init__.py

```diff
@@ -27,70 +28,118 @@
+            {
+                "name": "install_source",
+                "type": "STRING",
+                "description": "The source of a profile installation.",
+            },
+        ],
+    )
+    adjust = AttributionFieldGroup(
+        name="adjust",
+        source_pings=[Pings.metrics, Pings.first_session],
+        fields=[
+            {
+                "name": "adjust_ad_group",
+                "type": "STRING",
+                "description": "Adjust Ad Group the profile is attributed to.",
+            },
+            {
+                "name": "adjust_campaign",
+                "type": "STRING",
+                "description": "Adjust Campaign the profile is attributed to.",
+            },
+            {
+                "name": "adjust_creative",
+                "type": "STRING",
+                "description": "Adjust Creative the profile is attributed to.",
+            },
+            {
+                "name": "adjust_network",
+                "type": "STRING",
+                "description": "Adjust Network the profile is attributed to.",
+            },
+        ],
+    )
+    play_store = AttributionFieldGroup(
+        name="play_store",
+        source_pings=[Pings.first_session],
+        fields=[
+            {
+                "name": "play_store_attribution_campaign",
+                "type": "STRING",
+                "description": "Play store campaign the profile is attributed to.",
+            },
+            {
+                "name": "play_store_attribution_medium",
+                "type": "STRING",
+                "description": "Play store Medium the profile is attributed to.",
+            },
+            {
+                "name": "play_store_attribution_source",
+                "type": "STRING",
+                "description": "Play store source the profile is attributed to.",
+            },
+            # TODO: decide if this should be added here?
+            # {
+            #     "name": "play_store_attribution_install_referrer_response",
+            #     "type": "STRING",
+            #     "description": "Play store source the profile is attributed to.",
+            # },
+        ],
+    )
+    meta = AttributionFieldGroup(
+        name="meta",
+        source_pings=[Pings.first_session],
+        fields=[
+            {
+                "name": "meta_attribution_app",
+                "type": "STRING",
+                "description": "Facebook app linked to paid marketing.",
+            },
+        ],
+    )
+    is_suspicious_device_client = AttributionFieldGroup(
+        name="is_suspicious_device_client",
+        source_pings=[],
+        fields=[
+            {
+                "name": "is_suspicious_device_client",
+                "type": "BOOLEAN",
+                "description": "Flag to identify suspicious device users, see bug-1846554 for more info.",
+            },
+        ],
+    )
+    empty = AttributionFieldGroup(
+        name="empty",
+        source_pings=[],
```

Comment:
You could add an explicit NULL = "null" to the pings enumeration to use in this case:
```suggestion
        source_pings=pings.NULL,
```

Reply:
this is meant to be a list.